app/src/components/Card.css

```diff
@@ -0,0 +1,14 @@
+.card {
+  width: 376px;
+  height: 450px;
```

Comment:
do not use fixate height here. it is gonna be like this if you did the layout correct with all of the marigns and stuff

Reply:
lets fixate the width in `App.css` and `className` prop for the `Card` component